resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -722,8 +732,8 @@ private ApkAssetsCookie FindEntry(int resid, short density_override,
           ResTable_config this_config = ResTable_config.fromDtoH(type.config);
 
           if (this_config.match(desired_config)) {
-            if ((best_config == null || this_config.isBetterThan(best_config, desired_config)) ||
-                (package_is_overlay && this_config.compare(best_config) == 0)) {
+            if ((best_config == null || this_config.isBetterThan(best_config, desired_config))
+                || (package_is_overlay && this_config.compare(best_config) == 0)) {
```

Comment:
Are these formatting changes generated by google-java-format? If not, please keep the origin formatting if you don't change these lines.

Reply:
Yes, this is from google-java-format

I split the formatting changes into a separate commit in this PR to easily distinguish between the formatting changes and my new changes

Reply:
Thanks for your confirmation. I will double-check it locally again.